BambooTracker/gui/configuration_dialog.cpp

```diff
@@ -223,7 +223,7 @@ ConfigurationDialog::ConfigurationDialog(std::weak_ptr<Configuration> config, st
 		QObject::connect(button, &QToolButton::clicked, seq, &QKeySequenceEdit::clear);
 		auto layout = widget->layout();
 		layout->setSpacing(0);
-		layout->setMargin(0);
+		layout->setContentsMargins(0, 0, 0, 0);
```

Comment:
`QLayout::setMargin(int)` is obsoleted in Qt5 & was removed in Qt6, `QLayout::setContentsMargins(int, int, int, int)` or `QLayout::setContentsMargins(QMargins)` are the simple replacement options.